modules/ding_toggle_format/ding_toggle_format.module

```diff
@@ -117,7 +147,8 @@ function ding_toggle_format_preprocess(&$variables, $hook) {
   $toggle_format_pages = variable_get('ding_toggle_format_pages', array());
 
   foreach ($toggle_format_pages as $page) {
-    if (empty($page) || !strstr($_GET['q'], $page)) {
+    $user_pages = 'user' == $page && 'status' != arg(2);
```

Comment:
Please wrap the conditional into spaces. This changes nothing code-wise, but helps keep code visually more convenient.
`$user_pages = ('user' == $page && 'status' != arg(2));`